src/components/FilterPanel/FilterPanelSearch/__tests__/FilterSearch.spec.js

```diff
@@ -19,24 +19,24 @@ describe('FilterPanelSearch', () => {
   });
 
   it('renders results content when keyboard focus is added', () => {
-    const { getByTestId, getByLabelText } = render(
+    const { getByTestId, getAllByLabelText } = render(
       <FilterPanelSearch labelText="search label">
         <div data-testid="result-content" />
       </FilterPanelSearch>
     );
 
-    getByLabelText('search label').focus();
+    getAllByLabelText('search label')[0].focus();
     expect(getByTestId('result-content')).toBeVisible();
   });
 
   it('does not render results content when keyboard focus is removed', async () => {
-    const { queryByTestId, getByLabelText } = render(
+    const { queryByTestId, getAllByLabelText } = render(
       <FilterPanelSearch labelText="search label">
         <div data-testid="result-content" />
       </FilterPanelSearch>
     );
 
-    const searchInput = getByLabelText('search label');
+    const searchInput = getAllByLabelText('search label')[0];
     searchInput.focus();
     searchInput.blur();
```

Comment:
@diego-codes Just bringing this to your attention -- anything amiss in this change?